prettier.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  arrowParens: "always",
+  bracketSpacing: true,
+  embeddedLanguageFormatting: "auto",
+  htmlWhitespaceSensitivity: "css",
+  insertPragma: false,
+  bracketSameLine: false,
+  jsxSingleQuote: false,
+  printWidth: 80,
```

Comment:
:'(

Reply:
You can always overwrite it per-project ;d